types/lib/nodecg-instance.d.ts

```diff
@@ -0,0 +1,106 @@
+import {IRouter, RequestHandler} from 'express-serve-static-core';
+
+import {Logger} from './logger';
+import {ReplicantOptions, Replicant} from './replicant';
+import {NodeCGConfig} from './config';
+import {SendMessageReturnType} from './nodecg-static';
+import {Platform} from './platform';
+
+interface NodeCGCommon<P extends Platform, M = SendMessageReturnType<P>> {
+	bundleName: string;
+	bundleConfig: any;
+	bundleVersion: string;
+	readonly bundleGit: {
+		branch: string;
+		hash: string;
+		shortHash: string;
+		date?: Date;
+		message?: string;
+	};
+	Logger: typeof Logger;
+	log: Logger;
+	readonly config: NodeCGConfig;
+	sendMessage(
+		messageName: string,
+		cb?: (error: any, ...args: any[]) => void
+	): M;
+	sendMessage(
+		messageName: string,
+		data: any,
+		cb?: (error: any, ...args: any[]) => void
+	): M;
+	listenFor(messageName: string, handlerFunc: (message: any) => void): void;
+	listenFor(
+		messageName: string,
+		bundleName: string,
+		handlerFunc: (message: any) => void
+	): void;
+	unlisten(messageName: string, handlerFunc: (message: any) => void): void;
+	unlisten(
+		messageName: string,
+		bundleName: string,
+		handlerFunc: (message: any) => void
+	): void;
+	Replicant<V>(name: string, opts?: ReplicantOptions<V>): Replicant<V, P>;
+	Replicant<V>(
+		name: string,
+		namespace: string,
+		opts?: ReplicantOptions<V>
+	): Replicant<V, P>;
+}
+
+/**
+ * NodeCG instance in extensions
+ */
+export interface NodeCGServer extends NodeCGCommon<'server'> {
+	getSocketIOServer(): SocketIO.Server;
```

Comment:
This line causes this error for me during compilation:
```
../../types/lib/nodecg-instance.d.ts:56:23 - error TS2503: Cannot find namespace 'SocketIO'.

56  getSocketIOServer(): SocketIO.Server;
                         ~~~~~~~~
```

Reply:
Installing `@types/socket.io` and `@types/socket.io-client` from npm should fix it. They are included in package.json of this PR.